README.md

```diff
@@ -26,339 +26,212 @@
-
-* [Machine learning
-  frameworks](https://github.com/triton-inference-server/fil_backend).
-  Triton supports popular machine learning frameworks such as XGBoost,
-  LightGBM, Scikit-Learn and cuML using the [RAPIDS Forest Inference
-  Library](https://medium.com/rapids-ai/rapids-forest-inference-library-prediction-at-100-million-rows-per-second-19558890bc35).
-
-* [Concurrent model
-  execution](docs/architecture.md#concurrent-model-execution). Triton
-  can simultaneously run multiple models (or multiple instances of the
-  same model) using the same or different deep-learning and
-  machine-learning frameworks.
-
-* [Dynamic batching](docs/architecture.md#models-and-schedulers). For
-  models that support batching, Triton implements multiple scheduling
-  and batching algorithms that combine individual inference requests
-  together to improve inference throughput. These scheduling and
-  batching decisions are transparent to the client requesting
-  inference.
-
-* [Extensible
-  backends](https://github.com/triton-inference-server/backend). In
-  addition to deep-learning frameworks, Triton provides a *backend
-  API* that allows Triton to be extended with any model execution
-  logic implemented in
-  [Python](https://github.com/triton-inference-server/python_backend)
-  or
-  [C++](https://github.com/triton-inference-server/backend/blob/main/README.md#triton-backend-api),
-  while still benefiting from full CPU and GPU support, concurrent
-  execution, dynamic batching and other features provided by Triton.
-
-* Model pipelines using
+under-development progress towards the next release. The current release is 
+version [2.21.0](https://github.com/triton-inference-server/server/tree/r22.04)
+and corresponds to the 22.04 container release on 
+[NVIDIA GPU Cloud (NGC)](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/tritonserver).**
+
+----
+Triton Inference Server is an inference serving software that streamlines AI 
+inferencing. Triton enables teams to deploy any AI model from multiple deep 
+learning and machine learning frameworks, including TensorRT, TensorFlow, PyTorch, 
+ONNX, OpenVINO, Python and more. Triton supports inference across cloud, data center,
+edge and embedded devices on NVIDIA GPUs, x86 and ARM CPU, or AWS Inferentia. 
+Triton delivers optimized performance for many query types, including real time,
+batched, ensembles and audio/video streaming.
+
+Major features include:
+
+- [Multiple deep learning
+  framework support](https://github.com/triton-inference-server/backend)
+- [Multiple machine learning
+  framework support](https://github.com/triton-inference-server/fil_backend)
+- [Concurrent model
+  execution](docs/architecture.md#concurrent-model-execution)
+- [Dynamic batching](docs/model_configuration.md#dynamic-batcher)
+- [Sequence batching](docs/model_configuration.md#sequence-batcher) and 
+  [implicit state management](docs/architecture.md#implicit-state-management)
+- Provides [Extensible
```

Comment:
instead of provides extendible backends, can we say "Provides a backend API to allow adding custom backends and pre/post processing operations"